docs/pages/access-controls/reference.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Access Controls Reference Documentation
-description: Access Controls reference documentation for role options and properties.
+description: Learn how to use Teleport roles to manage role-based access controls (RBAC) in your Teleport cluster with this helpful guide.
```

Comment:
```suggestion
description: Learn how to use Teleport roles to manage role-based access controls (RBAC) in your Teleport cluster with this detailed guide.
```

Leaving it to the reader to determine if it's helpful

Reply:
I don't see this as an improvement. Do we have unhelpful (or undetailed) guides too? What stops us from just putting "with this helpful guide" on every single guide? Should we?

I am not an expert in SEO, but to me this just looks like extra words, and our general guidance thus far in docs has been to be clear and concise and avoid adding extra fluff.

I'm curious what @klizhentas thinks about this.

Reply:
The purpose behind these is to better control the meta descriptions, for example, what we currently have in `description: ` is ignored by search engines.  It instead picks snippets for us. These are mostly suggestions, I can rework them to remove the extra words. We ideally want meta descriptions that aren't concatenated. 

<img width="654" alt="image" src="https://user-images.githubusercontent.com/559288/229919386-79195c05-7de1-4560-a176-08541bddc7d4.png">
<img width="708" alt="image" src="https://user-images.githubusercontent.com/559288/229919480-be135dd0-672c-49dd-8316-81f2399e4087.png">